integrationtests/mayavi/common.py

```diff
@@ -461,6 +573,13 @@ def parse_command_line(self, argv):
         if options.verbose:
             VERBOSE = True
             self.log_mode = logging.DEBUG
+        global psutil
```

Comment:
Umm this is dirty, instead just perform the import in each function that requires it.  i.e. above line 106.